tests/Mono.Android-Tests/Mono.Android-Test.Shared.projitems

```diff
@@ -13,7 +13,6 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="$(MSBuildThisFileDirectory)System.Linq\LinqExpressionTest.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Resources\Resource.designer.cs" />
```

Comment:
You might have to make this conditional based on `$(AndroidUseIntermediateDesignerFile)`.